lms/static/sass/views/_teams.scss

```diff
@@ -252,6 +268,11 @@
         @include margin-right ($baseline*.75);
         color: $gray-d1;
 
+        abbr {
```

Comment:
I couldn't find this being used in the templates for this PR, where is this abbr used? cc'ed @frrrances